space2stats_api/app/utils/db_utils.py

```diff
@@ -0,0 +1,70 @@
+import os
+import psycopg2
+from dotenv import load_dotenv
+
+
+load_dotenv("../db.env")
+
+DB_HOST = os.getenv("DB_HOST")
+DB_PORT = os.getenv("DB_PORT")
+DB_NAME = os.getenv("DB_NAME")
+DB_USER = os.getenv("DB_USER")
+DB_PASSWORD = os.getenv("DB_PASSWORD")
+DB_TABLE_NAME = os.getenv("DB_TABLE_NAME")
+
+print(DB_TABLE_NAME)
```

Comment:
we could use Pydantic-settings https://github.com/developmentseed/tipg/blob/0975f181f28aa94e11ccf3bf8737cd8734421018/tipg/settings.py#L116-L159

Reply:
> we could use Pydantic-settings https://github.com/developmentseed/tipg/blob/0975f181f28aa94e11ccf3bf8737cd8734421018/tipg/settings.py#L116-L159

Implemented!